packages/extensions/core/resources/plugin-csharp.md

```diff
@@ -2,17 +2,25 @@
 
 The V2 version of the C# Generator.
 
-``` yaml $(csharp) && $(preview) && !isRequested('@autorest/csharp')
-# default the v2 generator to using the last stable @microsoft.azure/autorest-core 
+``` yaml $(csharp) && !$(legacy) && !$(v2) && !isRequested('@microsoft.azure/autorest.csharp')
+version: ~3.0.6298
+
+use-extension:
+  "@autorest/csharp": "^3.0.0"
```

Comment:
We should "latest" version of [@autorest/csharp](https://www.npmjs.com/package/@autorest/csharp) like [python](https://github.com/Azure/autorest/blob/master/packages/extensions/core/resources/plugin-python.md). Right @pakrym ?

Reply:
Yeah, I think we should be consistent.